crates/swc_ecma_minifier/tests/projects/output/mootools-1.4.5.js

```diff
@@ -3077,19 +3073,16 @@ Elements.prototype = {
         var to = {}, selectorTest = new RegExp("^" + selector.escapeRegExp() + "$");
         return Array.each(document.styleSheets, function(sheet, j) {
             var href = sheet.href;
-            if (!(href && href.contains("://")) || href.contains(document.domain)) {
-                var rules = sheet.rules || sheet.cssRules;
```

Comment:
Makes sense.

`var rules = sheet.rules || sheet.cssRules` is inlined into `Array.each(rules`